msg_startup_test.go

```diff
@@ -0,0 +1,184 @@
+package pgsrv
+
+import (
+	"encoding/binary"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestStartupVersion(t *testing.T) {
+	t.Run("typed message", func(t *testing.T) {
+		m := &msg{'p', 0, 0, 0, 5}
+		_, err := m.StartupVersion()
+
+		require.Error(t, err)
```

Comment:
@oshribin - this can be understood from test name.
however, yury, please add require.Equal for verifying error message here and for any other error test :)